bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java

```diff
@@ -1046,6 +1047,7 @@ private void replicateLedgerFragment(LedgerHandle lh,
         SingleFragmentCallback cb = new SingleFragmentCallback(
             resultCallBack,
             lh,
+            bkc.getMainWorkerPool(),
```

Comment:
@eolivelli as I said in another comment, this patch I only want to focus on the LedgerHandle injectability. This line change is just because it was previously using the raw field. In a later pass, i'd like to get rid of all the methods like getMainWorkerPool from bookkeeper completely, but doing all that now would make an already huge patch even bigger.

Reply:
Okay.
Patch looks good to me. Great work